### PR TITLE
cdc: add ticdc sql mode config

### PR DESCRIPTION
**Pull request by asddongmen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by CharlesCheung96 — APPROVED** [review]

**Review by sdojjy — APPROVED** [review]

**Review by nongfushanquan — APPROVED** [review]

File: ticdc/ticdc-ddl.md [hunk 1]
```diff
@@ -99,3 +99,26 @@ rules = ['test.t*']
...
+
+如果你未设置，那么 TiCDC 可能无法正确解析一些 DDL，例如：
+
+```sql
+CREATE TABLE "t1" ("a" int PRIMARY KEY);
+```
+
+因为在 TiDB 的默认 SQL 模式下，双引号会被视为字符串而不是标志符，这将会导致 TiCDC 无法正确解析该 DDL。
+
+因此，建议你在创建同步任务的时候，在配置文件中指定使用上游 TiDB 集群使用的 SQL 模式。
```
Comment (ran-huang) [inline ticdc/ticdc-ddl.md:124]:
```suggestion
因此，在创建同步任务的时候，建议在配置文件中指定使用上游 TiDB 集群使用的 SQL 模式。
```

**Review by ran-huang — COMMENTED** [review]
rest LGTM

File: ticdc/ticdc-ddl.md [hunk 2]
```diff
@@ -99,3 +99,26 @@ rules = ['test.t*']
 | `RENAME TABLE test.t1 TO test.ignore1, test.t3 TO test.ignore2` | 同步 | 旧的表库名，新的库名都符合 filter 规则 |
 | `RENAME TABLE test.t1 TO ignore.t1, test.t2 TO test.t22;` | 报错 | 新的库名 ignore 不符合 filter 规则 |
 | `RENAME TABLE test.t1 TO test.t4, test.t3 TO test.t1, test.t4 TO test.t3;` | 报错 | 在一条 DDL 中交换 test.t1 和 test.t3 两个表的名字，TiCDC 无法正确处理。请参考错误提示提示信息处理。 |
+
+### SQL 模式
```
Comment (Oreoxmt) [inline ticdc/ticdc-ddl.md:103]:
```suggestion
### SQL 模式
```

File: ticdc/ticdc-ddl.md [hunk 3]
```diff
@@ -99,3 +99,26 @@ rules = ['test.t*']
...
+
+如果未设置 SQL 模式，那么 TiCDC 可能无法正确解析一些 DDL 语句，例如：
+
+```sql
+CREATE TABLE "t1" ("a" int PRIMARY KEY);
+```
+
+因为在 TiDB 的默认 SQL 模式下，双引号会被视为字符串而不是标志符，这将会导致 TiCDC 无法正确解析该 DDL 语句。
+
+因此，在创建同步任务的时候，建议在配置文件中指定使用上游 TiDB 集群使用的 SQL 模式。
```
Comment (Oreoxmt) [inline ticdc/ticdc-ddl.md:124]:
```suggestion
因此，在创建同步任务的时候，建议在配置文件中指定使用上游 TiDB 集群设置的 SQL 模式。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**